src/humanpy/hum_kin1.py

```diff
@@ -0,0 +1,263 @@
+PACKAGE = 'humanpy'
```

Comment:
can you be clearer on the difference between hum_kin1 and hum_kin2? Those names are not super informative so it's no immediately clear.